packages/block-library/src/gallery/index.php

```diff
@@ -81,7 +81,7 @@ function block_core_gallery_render( $attributes, $content ) {
 	}
 
 	// Set the CSS variable to the column value, and the `gap` property to the combined gap value.
-	$style = '.' . $class . '{ --wp--style--unstable-gallery-gap: ' . $gap_column . '; gap: ' . $gap_value . '}';
+	$style = '.wp-block-gallery.' . $class . '{ --wp--style--unstable-gallery-gap: ' . $gap_column . '; gap: ' . $gap_value . '}';
```

Comment:
How might I test this? I can't find any way to change the gallery gap value via the UI 😅 
Would it be in theme.json?

Reply:
The gallery block is only supported at the individual block level (since it uses `skipSerialization` and its own custom handling), so it can be tested in the post (or site editor) by adding an individual Gallery block and adjusting the Block spacing under Dimensions. If I remember correctly, the block classname is needed here, so that the rule can override the base flex layout gap.